nl.lproj/Localizable.strings

```diff
@@ -0,0 +1,160 @@
+/* 
+  Localizable.strings
+  MobileRadio
+
+  Created by Steven Troughton-Smith on 01/11/2019.
+  Copyright © 2019 High Caffeine Content. All rights reserved.
+*/
+
+"SHORTCUT_PLAY" = "Speel";
+"SHORTCUT_PAUSE" = "Pauzeer";
+
+"TOOLBAR_NOTHING_PLAYING" = "Niets aan het spelen";
+"TOOLBAR_STREAMING_RADIO" = "Streaming Radio";
+
+"TOOLBAR_PLAYBUTTON_ACCESSIBILITY_LABEL" = "Speel/Pauzeer";
+"TOOLBAR_PLAYBUTTON_TOOLTIP" = "Speel of pauzeer";
+"TOOLBAR_MUTEBUTTON_TOOLTIP" = "Dempen aan of uit";
+"TOOLBAR_VOLUMESLIDER_ACCESSIBILITY_LABEL" = "Volume";
+
+"SIDEBAR_TITLE_LIBRARY" = "Bibliotheek";
+"SIDEBAR_TITLE_STATIONS" = "Verzamelingen";
+"SIDEBAR_TITLE_RADIO" = "Radio";
+
+"SIDEBAR_CATEGORY_ALL_STATIONS" = "Alle zenders";
+"SIDEBAR_CATEGORY_RECENTS" = "Recente";
+
+"LIST_HEADER_STATION" = "Zender";
+
+"MENU_NEW" = "Nieuw";
+"MENU_NEW_STATION" = "Zender…";
+"MENU_NEW_GROUP" = "Verzameling…";
+
+"DISCOVERABILITY_HUD_NEW_STATION" = "Nieuwe zender";
+"DISCOVERABILITY_HUD_NEW_GROUP" = "Nieuwe verzameling";
+
+"MENU_VIEW_AS_GRID" = "als raster";
+"MENU_VIEW_AS_LIST" = "als lijst";
+
+"DISCOVERABILITY_HUD_VIEW_AS_GRID" = "Bekijk als raster";
+"DISCOVERABILITY_HUD_VIEW_AS_LIST" = "Bekijk als lijst";
+
+"MENU_VIEW_SHOW_SIDEBAR" = "Toon navigatiekolom";
+"MENU_VIEW_HIDE_SIDEBAR" = "Verberg navigatiekolom";
+
+"MENU_HELP_GETSUPPORT" = "Krijg ondersteuning op Twitter…";
+"MENU_HELP_SHOW_WELCOME_SCREEN" = "Toon welkomstpaneel";
+
+"PLAYBACK_ERROR_STALL" = "De verbinding met de zender is verbroken.";
+
+"SEARCH_PLACEHOLDER" = "Zoek";
+
+"ALERT_OK" = "OK";
+"ALERT_ADD" = "Voeg toe";
+"ALERT_CANCEL" = "Annuleer";
+"ALERT_DELETE" = "Verwijder";
+
+"ADD_STATION_WINDOW_TITLE" = "Voeg nieuwe zender toe";
+"EDIT_STATION_WINDOW_TITLE" = "Wijzig zender";
+
+"EDIT_STATION_WINDOW_NAME_PLACEHOLDER" = "Naam";
+"EDIT_STATION_WINDOW_URL_PLACEHOLDER" = "http://pad/naar/stream";
+"EDIT_STATION_WINDOW_IMAGE_ACCESSIBILITY_LABEL" = "Kies afbeelding";
+
+"ADD_COLLECTION_WINDOW_TITLE" = "Voeg nieuwe verzameling toe";
+"EDIT_COLLECTION_WINDOW_TITLE" = "Wijzig verzameling";
+
+"EDIT_COLLECTION_WINDOW_NAME_PLACEHOLDER" = "Mijn verzameling";
+
+"SIDEBAR_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"SIDEBAR_CONTEXT_DELETE_COLLECTION" = "Verwijder verzameling";
+"SIDEBAR_CONTEXT_SHARE_COLLECTION" = "Deel";
+
+"STATION_CONTEXT_EDIT_COLLECTION" = "Wijzig";
+"STATION_CONTEXT_DELETE_COLLECTION" = "Verwijder zender";
+"STATION_CONTEXT_MOVE" = "Verplaats naar Verzameling";
+
+"CONTEXT_ALERT_DELETE_COLLECTION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_COLLECTION_BODY" = "Bent u zeker dat u deze collectie en alle zenders erin wenst te verwijderen?";
+
+"CONTEXT_ALERT_DELETE_STATION_TITLE" = "Verwijder \"%@\"";
+"CONTEXT_ALERT_DELETE_STATION_BODY" = "Bent u zeker dat u deze zender wenst te verwijderen?";
```

Comment:
"Weet u zeker dat u deze zender wenst te verwijderen?"

Reply:
> "Weet u zeker dat u deze zender wenst te verwijderen?"

Can I change it to “Weet u zeker dat u deze zender wilt verwijderen?", to make it more similar to the one above?